packages/store/addon/-private/system/record-arrays/adapter-populated-record-array.js

```diff
@@ -74,14 +74,46 @@ export default RecordArray.extend({
 
     return store._query(this.modelName, query, this);
   },
+});
+
+if (RECORD_ARRAY_MANAGER_IDENTIFIERS) {
+  /**
+    @method _setIdentifiers
+    @param {StableRecordIdentifier[]} identifiers
+    @param {Object} payload normalized payload
+    @internal
+  */
+  AdapterPopulatedRecordArray.prototype._setIdentifiers = function _setIdentifiers(identifiers, payload) {
```

Comment:
function setObjects(identifierOrInternalModel) {
    // TODO: initial load should not cause change events at all, only
    // subsequent. This requires changing the public api of adapter.query, but
    // hopefully we can do that soon.
    this.get('content').setObjects(identifiers);

    this.setProperties({
      isLoaded: true,
      isUpdating: false,
      meta: assign({}, payload.meta),
      links: assign({}, payload.links),
    });

    this.manager._associateWithRecordArray(identifiers, this);

    if (DEPRECATE_EVENTED_API_USAGE) {
      const _hasDidLoad = DEBUG ? this._has('didLoad') : this.has('didLoad');
      if (_hasDidLoad) {
        // TODO: should triggering didLoad event be the last action of the runLoop?
        once(this, 'trigger', 'didLoad');
      }
    }
  };
}

if (RECORD_ARRAY_MANAGER_IDENTIFIERS) {
  AdapterPopulatedRecordArray.prototype._setIdentifiers = function _setIdentifiers(identifiers, payload) {
    function setObjects(identifiers, payload)l
  } 
} else {
  AdapterPopulatedRecordArray.prototype._setInternalModels = function _setInternalModels(internalModels, payload) {
    function setObjects(identifiers, payload)l

}
}